packages/react/src/components/Selector/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import UiText from '../UiText';
+
+const Selector = React.forwardRef(
+  (
+    {
+      className,
+      disabled: isDisabled,
+      id,
+      onChange,
+      options,
+      defaultIndex,
+      ...props
+    },
+    ref,
+  ) => {
+    const [activeIndex, setActiveIndex] = useState(defaultIndex);
+
+    const handleOptionClick = (index, option) => {
+      if (isDisabled || option.disabled) return;
+      if (index === activeIndex) {
+        setActiveIndex(null);
+        onChange(null);
+        return;
+      }
+
+      setActiveIndex(index);
+      onChange(option.value);
+    };
+
+    return (
+      <div className='f-FormEl-wrapper'>
+        <UiText
+          as='div'
+          variant={UiText.VARIANTS.content}
```

Comment:
Not needed as it's the default prop
https://github.com/heetch/flamingo/blob/master/packages/react/src/components/UiText/index.js#L41